### PR TITLE
fix: API bug fixes

### PR DESCRIPTION
**Pull request by jjmaynard**
## Description
-Bug fixes for US soilID code

### CODE REVIEW

File: python_code/soilIDModel_US_v3.py [hunk 1]
```diff
@@ -819,16 +836,26 @@ def getSoilLocationBasedUS(lon, lat, plot_id):
...
+                [ilr2_l, ilr2_r, ilr2_h],
+                [row["dbovendry_l"], row["dbovendry_r"], row["dbovendry_h"]],
+                [row["wthirdbar_l"], row["wthirdbar_r"], row["wthirdbar_h"]],
+                [row["wfifteenbar_l"], row["wfifteenbar_r"], row["wfifteenbar_h"]],
+                [row["rfv_l"], row["rfv_r"], row["rfv_h"]],
+            ]
+
         # Check diagonal elements and off-diagonal range
-        if not np.all(np.diag(local_correlation_matrix) == 1) or np.any(
+        if not np.all(np.diag(local_correlation_matrix) >= 0.99999999999999) or np.any(
```
Comment (paulschreiber) [inline python_code/soilIDModel_US_v3.py:858]: suggestion: can you not use >= 1 here?

File: python_code/soilIDModel_US_v3.py [hunk 2]
```diff
@@ -756,11 +755,29 @@ def getSoilLocationBasedUS(lon, lat, plot_id):
         "dbovendry_r",
         "wthirdbar_r",
         "wfifteenbar_r",
-        "rfv_r",
+        # "rfv_r",
```
Comment (paulschreiber) [inline python_code/soilIDModel_US_v3.py:758]: suggestion: remove this if not in use.
```suggestion
```

**Review by paulschreiber — CHANGES_REQUESTED** [review]
@jjmaynard Please look over my two suggestions. Please also ensure you are running the current versions of `black` and `flake8` so we have consistent formatting.

File: python_code/soilIDModel_US_v3.py (hunk 1 above)
Comment (jjmaynard) [reply python_code/soilIDModel_US_v3.py:858]: In testing, I found cases where values were slightly less then 1, e.g., 0.999999999. This fixed the issue but more extensive testing is required.

**Review by paulschreiber — APPROVED** [review]

**Pull request merged**